dsp/window/window_test.go

```diff
@@ -119,7 +154,7 @@ func TestWindows(t *testing.T) {
 			dst := test.fn(src)
 
 			if !floats.EqualApprox(dst, test.want, tol) {
```

Comment:
It's not so critical here but in general when comparing two slices/vectors/matrices I prefer to get a measure of their distance and then compare this value to a tolerance. It's then easy to log the distance and see if a failure is due to too strict tolerance or an actual bug.

Reply:
Yeah. The slices here are short and we show them, so I'm OK with doing it this way.